diff --git a/meson.build b/meson.build
index 32406c2..56c000d 100644
--- a/meson.build
+++ b/meson.build
@@ -255,3 +255,8 @@ foreach plugin: plugins
   meson.override_dependency(plugin.name(), dep)
   gst_plugins += [dep]
 endforeach
+
+lzma_stub = dependency('', required: false)
+bz2_stub = dependency('', required: false)
+meson.override_dependency('lzma', lzma_stub)
+meson.override_dependency('bz2', bz2_stub)
